editor/plugins/script_editor_plugin.cpp

```diff
@@ -2538,11 +2538,7 @@ void ScriptEditor::reload_scripts(bool p_refresh_only) {
 
 			Ref<TextFile> text_file = edited_res;
 			if (text_file != nullptr) {
```

Comment:
I think this check is useless, `Ref<T>` is never nullptr afaik. It should use `text_file.is_valid()` to check the actual pointer.